package.json

```diff
@@ -65,6 +65,7 @@
     "eslint": "^6.7.2",
     "eslint-config-kentcdodds": "^14.6.1",
     "eslint-plugin-cypress": "^2.7.0",
+    "hasura-cli": "^1.0.0",
```

Comment:
This probably doesn't need to be in here, it'd maybe be a devDependency but seems like we'll just use it from our respective computers and the site itself doesn't need it.